python/lsst/daf/butler/datastore/cache_manager.py

```diff
@@ -1015,7 +1015,11 @@ class DatastoreDisabledCacheManager(AbstractDatastoreCacheManager):
         in lookup keys.
     """
 
-    def __init__(self, config: str | DatastoreCacheManagerConfig, universe: DimensionUniverse):
+    def __init__(
+        self,
+        config: str | DatastoreCacheManagerConfig | None = None,
+        universe: DimensionUniverse | None = None,
```

Comment:
This is the constructor for the no-op `DatastoreDisabledCacheManager` which doesn't actually use either of these parameters because it does nothing... but I think it has to have them as optionals because of the way the cache manager is instantiated from configuration.

Reply:
Oh yes. I had missed that it was the disabled cache constructor.